.github/workflows/test-objc.yaml

```diff
@@ -171,8 +173,22 @@ jobs:
         uses: actions/checkout@v2
       - name: Pull Carthage dependencies
         run: carthage bootstrap
-      - name: Build Carthage projects
-        run: carthage build --no-skip-current
+      - name: Build Carthage projects - excluding arm64
+        run: |
+          # ------ beginning of workaround
+          # from https://github.com/Carthage/Carthage/issues/3019#issuecomment-699143260
+          # supports Xcode 12.0.1
+          set -euo pipefail
+          xcconfig=$(mktemp /tmp/static.xcconfig.XXXXXX)
+          trap 'rm -f "$xcconfig"' INT TERM HUP EXIT
+          # For Xcode 12 make sure EXCLUDED_ARCHS is set to arm architectures otherwise
+          # the build will fail on lipo due to duplicate architectures.
+          echo 'EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64__XCODE_1200 = arm64 arm64e armv7 armv7s armv6 armv8' >> $xcconfig
+          echo 'EXCLUDED_ARCHS = $(inherited) $(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT)__XCODE_$(XCODE_VERSION_MAJOR))' >> $xcconfig
+          export XCODE_XCCONFIG_FILE="$xcconfig"          
+          # ------ end of workaround
```

Comment:
I believe it would be better to create a config file in our repo and then
```
export XCODE_XCCONFIG_FILE="path/to/our.xcconfig"
```
instead of creating a temporary file, etc.

The `set -euo pipefail` can also be omitted in this case. GitHub Actions already enable that, IIRC.

Reply:
I agree, but I really hope that these are temporally measures and we won't need them soon.
_(Nothing is more permanent than a temporary solution)_